src/components/Chips/Chips.tsx

```diff
@@ -66,13 +65,16 @@ interface ChipsProps extends VibeComponentProps {
    */
   ariaLabel?: string;
   /**
-   * Should element be focusable & clickable - for backward compatability
+   * Deprecated, there is no need to use this prop for implementing clickable chips. Please use onClick for this purpose.
+   * @deprecated
```

Comment:
noice!! maybe worth adding this for other props as well - WDYT?

Reply:
can be done :)